CHANGELOG.md

```diff
@@ -23,3 +23,28 @@ New modules
 
 Other minor additions
 ---------------------
+
+* Added new function in `Data.List.Base`:
+  ```agda
+  last : List A → Maybe A
+  ```
+
+* Added new proofs in `Data.List.Relation.Unary.All.Properties`:
+  ```agda
+  head⁺ : All P xs → All P (head xs)
```

Comment:
This doesn't make it clear at all the that rhs is `Maybe.All`. Without knowing that, the type signature looks bogus.